tests/Frontend/profilePic.spec.ts

```diff
@@ -0,0 +1,14 @@
+test("Users profile picture should display if present", () => {
+    const userImage = "https://i.scdn.co/image/ab6775700000ee85357ee0eabf98dc740c83bb9c"
+    const setProfilePic = userImage === undefined ? "https://i.ibb.co/WHfbS7L/logo.png" : userImage;
+    expect(setProfilePic).toBe(userImage);
```

Comment:
Nice one on starting out with Jest and testing! 👍

**No Action required** 

Right now, the test just checks if JavaScript and its ternary operators are doing their thing. It's not really looking at the component we're trying to test.

To focus on the component, you might want to test the actual DOM elements:

```js
import { render, screen } from '@testing-library/react';
import PersonalisedHomepage from '../PersonalisedHomepage';

it('should show the profile image', () => {
  render(<PersonalisedHomepage />);

  const imgElement = screen.getByRole('img');

  expect(imgElement).toBeInTheDocument();
  expect(imgElement).toHaveAttribute('src', 'expected-image-url');
});
```
To make this work, you'll need to add a package like `@testing-library/react` for DOM testing with Jest.

Here we are testing the front end and what the component renders in the document.

Reply:
This code is for demonstration purposes only.